packages/mco/components/modals/app-failover-relocate/parser/argo-application-set-parser.spec.tsx

```diff
@@ -0,0 +1,310 @@
+import * as React from 'react';
+import { DRActionType } from '@odf/mco/constants';
+import { DisasterRecoveryResourceKind } from '@odf/mco/hooks';
+import { fireEvent, render, screen, waitFor } from '@testing-library/react';
+// eslint-disable-next-line jest/no-mocks-import
+import {
+  mockDRClusterEast1,
+  mockDRClusterWest1,
+} from '../../../../__mocks__/drcluster';
+// eslint-disable-next-line jest/no-mocks-import
+import { mockDRPC1, mockDRPC3 } from '../../../../__mocks__/drplacementcontrol';
+// eslint-disable-next-line jest/no-mocks-import
+import { mockDRPolicy1 } from '../../../../__mocks__/drpolicy';
+// eslint-disable-next-line jest/no-mocks-import
+import {
+  mockManagedClusterEast1,
+  mockManagedClusterWest1,
+  mockManagedClusterWest1Down,
+} from '../../../../__mocks__/managedcluster';
```

Comment:
You can disable an eslint rule on a block basis (no need to disable it line by line)
```suggestion
/* eslint-disable jest/no-mocks-import */
import {
  mockDRClusterEast1,
  mockDRClusterWest1,
} from '../../../../__mocks__/drcluster';
import { mockDRPC1, mockDRPC3 } from '../../../../__mocks__/drplacementcontrol';
import { mockDRPolicy1 } from '../../../../__mocks__/drpolicy';
import {
  mockManagedClusterEast1,
  mockManagedClusterWest1,
  mockManagedClusterWest1Down,
} from '../../../../__mocks__/managedcluster';
/* eslint-enable jest/no-mocks-import */
```

Reply:
BTW, if we find that an eslint rule is not helpful, we can turn it off if there is consensus.